crates/node-core/src/args/txpool_args.rs

```diff
@@ -44,6 +44,11 @@ pub struct TxPoolArgs {
     /// Price bump percentage to replace an already existing blob transaction
     #[arg(long = "blobpool.pricebump", default_value_t = REPLACE_BLOB_PRICE_BUMP)]
     pub blob_transaction_price_bump: u128,
+
+    /// Maximum size a single transaction can have
```

Comment:
this should mention pool and bytes,
this is the max size in bytes allowed to enter the pool